transformer_engine/common/recipe/__init__.py

```diff
@@ -351,3 +356,35 @@ def __repr__(self) -> str:
             f"fp8_dpa={self.fp8_dpa}, "
             f"fp8_mha={self.fp8_mha}"
         )
+
+
+@dataclass()
+class QLinearParams:
```

Comment:
We should make it more obvious that this is specific to `CustomRecipe`. The name makes me think it is similar to `MMParams` or `QParams`:
https://github.com/NVIDIA/TransformerEngine/blob/9c67da6e8ca42c2f7627cde4bbd87e9f0569c865/transformer_engine/common/recipe/__init__.py#L44-L62
Maybe:
```suggestion
class CustomRecipeLinearQuantizers:
```

Reply:
Renamed to `CustomQuantizerFactories`.